### ISSUE TITLE
Would you recommend using or finetuning this model?

### ISSUE THREAD

**Issue by fire** [seal-rg/recurrent-pretraining#2]
> Our checkpoint is trained for only 47000 steps on a broadly untested data mixture with a constant learning rate. As an academic project, the model is trained only on publicly available data and the 800B token count, while large in comparison to older fully open-source models such as the Pythia series, is small in comparison to modern open-source efforts such as OLMo, and tiny in comparison to the datasets used to train industrial open-weight models.

Would you recommend using or finetuning this model or is it more a academic test report?

**Comment by JonasGeiping** [comment]
It's an interesting open question right now how best to finetune this kind of model. 

I think that that's pretty exciting, but to be reasonable for e.g. a product, there is no good reason to use this particular model over an industrial open-weight model like llama or qwen as of today, if you do not have a particular usecase that leverages the unique features of this model.

**Comment by fire** [comment]
I have a problem called meshgpt for text-to-3d and there multiple dimensions of the problem. 1. context length 2. the mesh encoding tokens limit of 1300 triangles 3. better performance (cot, etc) 4. watertight 3d manufacturable shapes. So was researching.

**Comment by rawwerks** [comment]
hey @fire - we should link up! i'm also really interested in text-to-CAD and text-to-3D, and part of what got me excited about this model was that so much of 3d design cannot be easily captured in words.

**Comment by fire** [comment]
I messaged you.

Did you know I integrated https://github.com/elalish/manifold into the next version of Godot Engine 4.4 for manifold mesh booleans?

![Mesh booleans](https://github.com/user-attachments/assets/5e67f3a1-ae17-4a13-86a8-171b7feba01b)

**Comment by JonasGeiping** [comment]
Closing this for now. For more questions, we can also follow up in the other finetuning question https://github.com/seal-rg/recurrent-pretraining/issues/3.
